antsibull/data/docsite/plugins_by_collection.rst.j2

```diff
@@ -2,6 +2,12 @@
 :orphan:
 {% endif %}
 
+{% macro list_plugins(plugin_type) %}
+{%   for name, desc in plugin_maps[plugin_type].items() | sort %}
+* :ref:`@{ name }@ <ansible_collections.@{ collection_name }@.@{ name }@_@{ plugin_type }@>` -- @{ desc | rst_ify | indent(width=2) }@
+{%   endfor %}
```

Comment:
I know this was sort of moved from another place so it's just doing what was already there, but what is the extra spacing for inside `{%    ` on some elements? Does that control indentation somehow?

Reply:
It's basically a visual help for readers. You can see that this is part of another block. (Indenting the `{%` itself is not a good idea since that results in extra whitespace in the output.)

Reply:
As an alternative to whitespace inside the brackets, it's also possible to use `{%-` and `-%}` to suppress the extra whitespace.

Reply:
@tremble true, but the `-` quickly eats too much whitespace (like newlines as well).